src/app/services/user/helpers/account-axies.ts

```diff
@@ -97,7 +98,7 @@ export class AccountAxies {
         const queryOutput: any = await this._apollo.query({
           query: GET_PROFILE_AXIES_BY_RONIN_ADDRESS,
           variables: variables,
-        }).toPromise();
+        }).pipe(retry(3)).toPromise();

```

Comment:
Ah cool, we can do this in the pipe. Maybe can we use the retryWhen to make a do a exponential back of. e.g. have a look at example 2 in https://www.learnrxjs.io/learn-rxjs/operators/error_handling/retrywhen

Reply:
I have added the retry strategy, please check it and let me know if you want to change it